travis/test_server.py

```diff
@@ -136,8 +137,13 @@ def get_addons_path(travis_dependencies_dir, travis_build_dir, server_path):
     return addons_path
 
 
-def get_server_script(odoo_version):
-    return 'odoo-bin' if float(odoo_version) >= 10 else 'openerp-server'
+def get_server_script(server_path):
+    try:
+        shutil.copy(os.path.join(server_path, 'openerp-server'),
+                    os.path.join(server_path, 'odoo-bin'))
+    except IOError:
```

Comment:
It's best for the `get_server_script` to not have side effects.
You should instead check if `odoo-bin` exists, and fall back to `openep-server` if not.